blockstack-vm/src/functions/lists.rs

```diff
@@ -0,0 +1,87 @@
+use super::InterpreterResult;
+use super::super::errors::Error;
+use super::super::types::{Value, TypeSignature};
+use super::super::types::Value::{List};
+use super::super::representations::SymbolicExpression;
+use super::super::representations::SymbolicExpression::{AtomValue};
+use super::super::{Context,Environment,eval,apply,lookup_function};
+
+pub fn list_cons(args: &[Value]) -> InterpreterResult {
```

Comment:
Is there an upper bound on how long a list can be?

Reply:
Right now, no, but I think we should let the "maximum value size" enforce this (as in, if we have a maximum value size of 1000 bytes, then you cannot have a list of longer than 1000/16 integers, e.g.)